tests/errmsgs/tundeclared_symbol.nim

```diff
@@ -0,0 +1,15 @@
+discard """
+  cmd: '''nim check --hints:off $file'''
+  action: reject
+  nimout: '''
+tundeclared_symbol.nim(12, 17) Error: symbol used before declaration: 'c' [generated in tundeclared_symbol.nim(12, 17)]
+'''
```

Comment:
```suggestion
  errormsg: "symbol used before declaration: 'c' [generated in tundeclared_symbol.nim(9, 17)]"
  line: 9
```
This is more in line with the other tests from `errmsgs`. I've also modified the line numbers, so applying the change should just work.